release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -80,21 +80,29 @@ For more information, see xref:../backup_and_restore/hibernating-cluster.adoc#hi
 [id="ocp-release-notes-extensions_{context}"]
 === Extensions ({olmv1})
 
-Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release and has helped grow and enable a substantial ecosystem of solutions and advanced workloads running as Operators. {product-title} {product-version} introduces {olmv1}, the next-generation Operator Lifecycle Manager, as a General Availability feature, designed to improve how you manage Operators on {product-title}.
+[id="ocp-release-notes-extensions-olmv1-ga_{context}"]
+==== {olmv1-first} (General Availability)
 
-Previously available as a Technology Preview feature only, this updated framework evolves many of the concepts that have been part of previous versions of {olmv0} by simplifying Operator management, enhancing secrutiy, and boosting reliability.
+Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release and has helped enable and grow a substantial ecosystem of solutions and advanced workloads running as Operators. {product-title} {product-version} introduces _{olmv1}_, the next-generation Operator Lifecycle Manager, as a General Availability (GA) feature, designed to improve how you manage Operators on {product-title}.
 
 [NOTE]
 ====
-The documentation for {olmv1} exists as a stand-alone guide called xref:../extensions/index.adoc#olmv1-about[Extensions]. Previously, {olmv1} documentation was a subsection of the xref:../operators/index.adoc#operators-overview[Operators] guide, which otherwise documents the {olmv0} feature set.
+With {olmv1} now generally available, starting in {product-title} {product-version} the existing version of OLM that has been included since the launch of {product-title} 4 is now known as _{olmv0}_.
+====
 
-The updated location and guide name reflect a more focused documentation experience and aims to differentiate between {olmv1} and {olmv0}.
+Previously available as a Technology Preview feature only, the updated framework in {olmv1} evolves many of the concepts that have been part of {olmv0} by simplifying Operator management, enhancing secrutiy, and boosting reliability.
+
+[NOTE]
+====
+Starting in {product-title} {product-version}, {olmv1} is now enabled by default, alongside {olmv0}. Cluster administrators can toggle installation of {olmv1} by using cluster capabilities at installation time.
```

Comment:
I have commented on this in the older version of the PR, hence commenting here again . We should say that OLMV1 is a capability which users can disable during installation Cluster capabilities

Reply:
Updated to:
```
[IMPORTANT]
====
* Starting in {product-title} {product-version}, {olmv1} is now enabled by default, alongside {olmv0}. {olmv1} is a xref:../installing/overview/cluster-capabilities.adoc#cluster-operators-ref-olmv1_cluster-capabilities[cluster capability] that administrators can optionally disable before installation of {product-title}.
* {olmv0} remains fully supported throughout the {product-title} 4 lifecycle.
====
```